src/nodelet/face_recognition_nodelet.cpp

```diff
@@ -474,7 +474,15 @@ namespace face_recognition {
 
       if (config_.model_threshold != config.model_threshold) {
         try {
+#if CV_MAJOR_VERSION >= 3
+          if(face::BasicFaceRecognizer* model = dynamic_cast<face::BasicFaceRecognizer*>(model_.get())) {
```

Comment:
This is necessary until https://github.com/opencv/opencv_contrib/pull/797 is merged and released.